### ISSUE TITLE
Document QARTOD / add to ncei packet

### ISSUE THREAD

**Issue by kkoch** [glos/myglos#178]
Need to document QARTOD and add to ncei.  Will need to look at the metadata and make sure it is linked appropriately.

**Comment by kkoch** [comment]
@kknee I've added the two thredds links for habs and buoys to the QARTOD documentation page (on our Doc Wiki).  Do you have other documentation we can add there?  

https://github.com/glos/Documentation/wiki/QARTOD-(DMAC)

**Comment by kknee** [comment]
@kkoch do we want to mention the packages that we used for application:
https://github.com/ioos/qartod + https://github.com/glos/glos-qartod

How about documenting the tests were're running and the test parameters used on there too?

**Comment by kkoch** [comment]
@kknee Yes! definitely.  I will add the packages.  Would love it if we have the additional documentation you mention as well.  Do you already have something available or is this something we need to get approvals to develop?

I added the links to the IOOS documentation and QARTOD manuals as well.

**Comment by kkoch** [comment]
per DMAC meeting @kknee yes pull together the thresholds and parameters documentation when you have a chance.  You can either send it to me or you can put directly on the wiki.

**Comment by kknee** [comment]
@kkoch just emailed you a spreadsheet of variables, tests, and test parameters/thresholds.

**Comment by gcutrell** [comment]
@tslawecki @kknee There is a script running on process1 (/usr/local/glos/obs2nc) that runs on the first day of the month that pulls together the last month of data for each station and places them in an ftp directory as a tar.gz. I'm guessing NCEI has a script to retrieve data from the ftp directory. They must dynamically add new stations and keep track if there is data after each month because I get emails each month whether a station does/doesn't have data. As long as the the netcdf files are NODC NC compliant we should be able to send them the quartod data. Like Kelly mentioned we probably need to coordinate with NCEI about the Quartod files.
